Firestore/core/src/firebase/firestore/util/path.cc

```diff
@@ -16,10 +16,41 @@
 
 #include "Firestore/core/src/firebase/firestore/util/path.h"
 
+#include "absl/strings/ascii.h"
+#include "absl/strings/string_view.h"
+
 namespace firebase {
 namespace firestore {
 namespace util {
 
+namespace {
+
+#if defined(_WIN32)  // not yet used elsewhere
+static constexpr absl::string_view::size_type npos = absl::string_view::npos;
```

Comment:
Question: why is this necessary? (I'm probably missing some context)

Reply:
Apparently I should have removed this altogether. Instead I've pull a commit forward that just uses it to reduce noise at the call sites.